eng/Packages.Data.props

```diff
@@ -195,6 +198,7 @@
     <PackageReference Update="Microsoft.AspNetCore.Server.Kestrel" Version="2.1.3" />
     <PackageReference Update="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.1.25" />
     <PackageReference Update="Microsoft.AspNetCore.Server.WebListener" Version="1.1.4" />
+    <PackageReference Update="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="1.1.5" />
```

Comment:
This version of the package is more than 3 years old and is officially deprecated.   If we need this reference, why are we not using the latest?

Reply:
Changes reverted.